src/webview/html/template.js

```diff
@@ -14,7 +14,7 @@ import escape from 'lodash.escape';
  *   template`Hello $\!${&<world}` -> 'Hello $!&amp;&lt;world'
  */
 export default (strings: string[], ...values: Array<string | number>) => {
-  // $FlowFixMe github.com/facebook/flow/issues/2616
+  // $FlowFixMe[prop-missing] github.com/facebook/flow/issues/2616
```

Comment:
Separately: this change in Flow called my attention to a feature/convention that's long existed, and would be handy to use here!

> `$FlowIssue`: for a type error that you suspect is an issue with Flow

https://flow.org/en/docs/errors/

No rush to do this at the same time as adding the error codes, but probably we should use `$FlowIssue` for at least the cases where we actually link to an issue in the Flow tracker :slightly_smiling_face: